extensions/wikia/CommunityPage/templates/moderatorsModule.mustache

```diff
@@ -0,0 +1,21 @@
+{{#topModeratorsData}}
+    {{#topModeratorsModuleEnabled}}
+    <div class="module moderators-module">
```

Comment:
can we use `.community-page-rail__ moderators-module` class? It's a step toward BEM convention and will be consistent with rest of the specific modules in this extension